### PR TITLE
New Module: Na_ontap_flexcache

### PR DESCRIPTION
**Pull request by carchi8py**
##### SUMMARY
Add the ability to manage flexcache

##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME
* na_ontap_flexcache

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by thedoubl3j** [comment]
cc @thaumos

**Review by beeankha — APPROVED** [review]
👍

**Review by lonico — APPROVED** [review]

**Review by lonico — APPROVED** [review]
shipit

File: lib/ansible/modules/storage/netapp/na_ontap_flexcache.py [hunk 1]
```diff
@@ -0,0 +1,475 @@
...
+class NetAppONTAPFlexCache(object):
+    """
+    Class with FlexCache methods
+    """
+
+    def __init__(self):
+
+        self.argument_spec = netapp_utils.na_ontap_host_argument_spec()
+        self.argument_spec.update(dict(
+            state=dict(required=False, type='str', choices=['present', 'absent'],
```
Comment (gundalow) [inline lib/ansible/modules/storage/netapp/na_ontap_flexcache.py:136]: Note for the future `required=False` is the default, so you can omit it.

**Review by gundalow — APPROVED** [review]

**Pull request merged**